README.md

```diff
@@ -60,7 +61,7 @@ weather === "sunny";
 What are the values of these expressions?  Explain your answers.
 
 <!-- Replace this comment with your answer -->
-
+weather = true.  the === asks if weather is the same as the variable.  both are true.
```

Comment:
No, this is not true. Also, the question was specifically asking what each line evaluated to.

Line 1, `var weather;`, evaluates to `undefined`.
Line 2, `weather = "sunny"`, evaluates to `"sunny"` because the `=` operator always evaluates to the value that it's trying to assign.
Line 3, `weather === "sunny"`, evaluates to `true` because it's comparing `weather` and `"sunny"`.